guides/common/modules/proc_configuring-alternate-content-sources.adoc

```diff
@@ -0,0 +1,54 @@
+[id="Configuring_Alternate_Content_Sources_{context}"]
+= Configuring Alternate Content Sources
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Content* > *Alternate Content Sources* > *Add Source*.
+. Set the *Source type* as *Custom*.
+. Select the *Content type*.
+. Enter a name in the *Name* field and optionally provide a description in the *Description* field.
```

Comment:
Please split this line. You can use `git grep` to see examples on how we handle entering optional descriptions to various entities.